run/markdown-preview/editor/test/runner.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail;
+
+requireEnv() {
+  test "${!1}" || (echo "Environment Variable '$1' not found" && exit 1)
+}
+requireEnv SERVICE_NAME
+
+# The markdown-preview sample needs to be tested with both the editor and renderer services deployed.
+echo '---'
+
+# Build the Renderer service.
+test/service.sh
```

Comment:
```suggestion
export UPSTREAM_SERVICE_NAME=$(test/service.sh)
```

Reply:
if I export UPSTREAM_SERVICE_NAME here, do I still need to requireEnv UPSTREAM_SERVICE_NAME on the next line?

Reply:
It doesn't hurt to keep it because it will also check that it isn't empty